pivotal-import/.gitignore

```diff
@@ -4,3 +4,6 @@ data/shortcut_users.csv
 data/states.csv
 data/users.csv
 config.json
+
+# Emacs tmp files
+*\~
```

Comment:
I think it's fine to have this in this repo's `.gitignore`, but I keep these kinds of things in a global ignore file personally. You can add this to your `~/.gitconfig`

```
[core]
    excludesfile = "/Users/dlg/.gitignore_global"
```

And then whatever file you specify as your `excludesfile` has things like temporary files, compiled output formats, etc.